backend/src/api/metadata.ts

```diff
@@ -14,6 +14,13 @@ export const saveMetadata = async (cid: string, metadata: OffchainMetadata) => {
   return metadataRepository.setMetadata(cid, metadata);
 };
 
+export const searchMetadataByCID = async (cid: string, limit?: number) => {
```

Comment:
Just trying to simplify
```suggestion
export const searchMetadataByCID = async (cid: string, limit: number =  5) => {
```